doc/api/domain.md

```diff
@@ -25,6 +25,8 @@ will be notified, rather than losing the context of the error in the
 `process.on('uncaughtException')` handler, or causing the program to
 exit immediately with an error code.
 
+*Note*: This module is not available in [`Worker`][]s.
```

Comment:
So to clarify, if a worker throws what should happen?

Reply:
You get an `error` event in the main thread, which I now realize is missing from the documentation.